### PR TITLE
[WIP] Add jalires file from @jalires

### PR DESCRIPTION
**Pull request by jalires**
:warning:  Still things to add

## TODO
- [ ] Favorite hometown restaurant

@githubteacher for your approval

### CODE REVIEW

**Comment by jalires** [comment]
cc @sarafrederixon

File: jalires.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Hello!
+
+## Info:
+- Name: James Alires
+- Hometown: Las Vegas, NV
+- Favorite winter activity: Extreme snowball fights
```
Comment (sarafrederixon) [inline jalires.md]: :+1:

Comment (sarafrederixon) [inline jalires.md]: http://data3.whicdn.com/images/49190609/large.gif

**Comment by sarafrederixon** [comment]
image

**Comment by mattmunich** [comment]
:+1:

**Pull request closed without merging**